recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -45,6 +46,7 @@ PACKAGECONFIG ?= " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'widevine',             'opencdmi_wv', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'thunder',              'network', '', d)} \
     ${@bb.utils.contains('MACHINE_FEATURES', 'wifi',                'network wifi', '', d)} \
+    ${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'none', '', 'firmwarecontrol', d)} \
```

Comment:
I would go with enabling firmware control based on machine/distro features rather than on the FIRMWARECONTROL_PLATFORM flag. I think it should be set on a machine level